newton/tests/test_softbody.py

```
@@ -0,0 +1,509 @@
+                test.assertNotEqual(
+                    colors[v0],
+                    colors[v1],
+                    f"vertices {v0} and {v1} share color {colors[v0]}",
+                )
+
+
+class TestSoftBody(unittest.TestCase):
+    pass
+
+
+def test_tet_adjacency_single_tet(test, device, solver):
```

Ruff (0.14.14)

Unused function argument: `test`

(ARG001)